tests/test_fin_builder.py

```diff
@@ -86,30 +86,31 @@ def test_fin_builder():
   args = GoFishArgs()
   machine_lst = load_machines(args)
   machine = machine_lst[0]
-
-  args.session_id = add_test_session()
+  args.label = 'tuna_pytest_fin_builder'
+  args.session_id = add_test_session(label='tuna_pytest_fin_builder')
+  miopen = MIOpen()
+  miopen.args = args
 
   #update solvers
-  kwargs = get_worker_args(args, machine)
+  kwargs = get_worker_args(args, machine, miopen)
```

Comment:
this is just a helper in tests. This function does not exist in /tuna. There is no point in making this a class function since there are some more steps in between in each miopen/example lib.